tsconfig.json

```diff
@@ -17,5 +17,5 @@
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true
   },
-  "include": ["src/**/*"]
+  "include": ["src/**/*", "batching-toposort.d.ts"]
```

Comment:
Try this package instead with native ts support?
https://github.com/n1ru4l/toposort